lms/static/lms/js/require-config.js

```diff
@@ -200,6 +204,19 @@
             },
             "moment-with-locales": {
                 exports: "moment"
+            },
+            "modernizr": {
+                exports: "Modernizr"
+            },
+            "afontgarde": {
+                exports: "AFontGarde"
+            },
+            "edxicons": {
```

Comment:
See notes above.

The tricky case will be to make sure that in the LMS, the optimizer is able to handle the way that edxicons and draggabilly are checking for the presence of RequireJS. It has to be a very specific syntax, or else the optimizer is not able to modify the check to work with our namespaced version of RequireJS (specific to the LMS). If this doesn't work properly, you will see those vendor files appearing individually (and most likely not cached) on the sandbox.

If this is an issue, @andy-armstrong or I can help.